medusa/process_tv.py

```diff
@@ -453,26 +457,17 @@ def unrar(self, path, rar_files, force=False):
                             break
 
                     if not skip_extraction:
-                        rar_handle.extract(path=path, withSubpath=False, overwrite=False)
+                        rar_handle.extract(path)
```

Comment:
When you code something (this is a general rule), you never do it with the assumption that something that you have no control over will never change. Thing evolve and change constantly. Just because it works today, it doesn't mean it will work tomorrow. This way of thinking is very shortsighted.